.github/workflows/ci-python-sdk.yml

```diff
@@ -1,14 +1,10 @@
 name: Python SDK CI
 
 on:
-  push:
-    paths:
-      - 'lib/python-sdk/**'
-      - '.github/workflows/ci-python-sdk.yml'
   pull_request:
     paths:
-      - 'lib/python-sdk/**'
-      - '.github/workflows/ci-python-sdk.yml'
```

Comment:
Removes the `push` trigger because it was causing duplicate runs of the CI checks on each PR. Not a big deal, but HHS admins recently asked us to try to reduce GitHub action usage wherever we can.